README.md

```diff
@@ -194,6 +194,7 @@ but only with a `url` must provide a `checksum` that matches the existing plugin
 | [`selectionhighlight`](plugins/selectionhighlight.lua?raw=1)                                                   | Highlights regions of code that match the current selection *([screenshot](https://user-images.githubusercontent.com/3920290/80710883-5f597c80-8ae7-11ea-97f0-76dfacc08439.png))*                                                                                                                |
 | [`settings`](plugins/settings.lua?raw=1)                                                                       | Provides a GUI to manage core and plugin settings, bindings and select color theme *([video](https://user-images.githubusercontent.com/1702572/169743674-ececae24-f6b7-4ff2-bfa2-c4762cd327d9.mp4))*. (depends on [`widget`](https://github.com/lite-xl/lite-xl-widgets))                        |
 | [`smallclock`](plugins/smallclock.lua?raw=1)                                                                   | Displays the current time in the corner of the status view                                                                                                                                                                                                                                       |
+| [`smartopenselected`](plugins/smartopenselected.lua?raw=1)                                                     | Opens the selected filename or path in project. Useful to open imports.                                                                                                                                                                                                                          |
```

Comment:
You can remove this change as it will be automatically generated from the manifest.